server/service/integration_mdm_dep_test.go

```diff
@@ -36,9 +39,35 @@ type profileAssignmentReq struct {
 	Devices     []string `json:"devices"`
 }
 
+func (s *integrationMDMTestSuite) depSetup(ctx context.Context, serverURL string) {
+	t := s.T()
+
+	depSvc := apple_mdm.NewDEPService(s.ds, s.depStorage, s.logger)
+	require.NoError(t, depSvc.CreateDefaultAutomaticProfile(ctx))
+	mysql.CreateABMKeyCertIfNotExists(t, s.ds)
+	mysql.CreateAndSetABMToken(t, s.ds, "fleet")
+	err := s.depStorage.StoreConfig(ctx, "fleet", &nanodep_client.Config{BaseURL: serverURL})
+	require.NoError(t, err)
+}
+
 func (s *integrationMDMTestSuite) TestDEPEnrollReleaseDeviceGlobal() {
 	t := s.T()
 	ctx := context.Background()
+	// Set up a mock Apple DEP API
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
two notes:

1. for the integration tests please use `*integrationMDMTestSuite.mockDEPResponse` which does all the setup for you (please remove `depSetup` as well)
2. for the org name, instead of `"fleet"` use `t.Name()` or a random value, this implicitly helps to prevent tests interfering with each other and implicitly tests multiple tokens.